tests/xml/XMLBearTest.py

```diff
@@ -18,14 +20,59 @@
 <a>hey and hi</a>
 """.splitlines(keepends=True)
 
-InvalidXMLCorrectedTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_file,),
-                                        invalid_files=(invalid_xml_file,),
-                                        tempfile_kwargs={"suffix": ".xml"})
-
-InvalidXMLSyntaxTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_chars,),
-                                        invalid_files=(invalid_xml_chars,),
-                                        tempfile_kwargs={"suffix": ".xml"})
+dtd_file = os.path.join(os.path.dirname(__file__),
+                        "test_files",
+                        "note.dtd")
+
+schema_file = os.path.join(os.path.dirname(__file__),
+                           "test_files",
+                           "note.xsd")
+
+valid_xml_path = list(open(os.path.join(os.path.dirname(__file__),
```

Comment:
@vivek425ster you can always split lines I think ? most languages (including html/xml) have method of limiting lines to 80 chars.